### ISSUE TITLE
Is the tag manager already setup for the demo account?

### ISSUE THREAD

**Issue by akshayreddykotha** [dwyl/learn-google-analytics#25]
Hi there,

The README.md is very helpful to dive in from scratch into setting up GA. My question is whether the tag manager is already setup for the [demo account](https://support.google.com/analytics/answer/6367342?hl=en)?

Is that the reason I already see hits when I access the demo account of google analytics?

Regards
Akshay

**Comment by nelsonic** [comment]
@akshayreddykotha yes, typically the demo account already has everything setup so it's ready to use. 👍

**Comment by akshayreddykotha** [comment]
Thanks @nelsonic. Few follow up questions though: 

1. What is the highest level thing I can do with that access?

2. Vaguely, I can say that I can create some custom reports and dashboards. But can I any advanced stuff with respect to analysis or insight generation?

Thanks!
Akshay

P.S. I understand that insight generation depends solely on how the data is interpreted by the analyst but you can consider my questions from the 'features of GA' perspective.
